### ISSUE TITLE
Java not found ( 1.8 )

### ISSUE THREAD

**Issue by aowss** [asciidoctor/docker-asciidoctor#215]
Hi,

i get the following error `Failed to load AsciiDoc document - Could not find Java executable (RuntimeError)` when running the following command `asciidoctor -r asciidoctor-diagram *.adoc` with the `1.8` image.

This doesn't happen with `1.7`.

Thanks for your help

**Comment by dduportal** [comment]
Hello @aowss , thansk for reporting! 

Since the test harness does not have this error, and I can't reproduce it:

- Test code: https://github.com/asciidoctor/docker-asciidoctor/blob/main/tests/asciidoctor.bats#L104
- The adoc file being used: https://github.com/asciidoctor/docker-asciidoctor/blob/main/tests/fixtures/basic-example.adoc
- The last test harness execution (with the `1.8.1` tag): https://github.com/asciidoctor/docker-asciidoctor/runs/3112321301?check_suite_focus=true#step:9:25
- OpenJDK 8 is installed and working with the latest image:

```shell
docker run --rm --entrypoint=which asciidoctor/docker-asciidoctor:1.8.1 java
/usr/bin/java
➜  charts git:(8c00032) docker
➜  charts git:(8c00032) docker run --rm --entrypoint=java asciidoctor/docker-asciidoctor:1.8.1 -version
openjdk version "1.8.0_282"
OpenJDK Runtime Environment (IcedTea 3.18.0) (Alpine 8.282.08-r1)
OpenJDK 64-Bit Server VM (build 25.282-b08, mixed mode)
```

It means that there a case in your setup that triggers this error: could you share the command and files used ?

**Comment by aowss** [comment]
I don't think this is part of my setup since the error occurs on our pipeline and the only think i had to do to fix it is change `image: asciidoctor/docker-asciidoctor` to `image: asciidoctor/docker-asciidoctor:1.7.0`

**Comment by dduportal** [comment]
Hello @aowss , I'm not saying that you did something wrong: I'm asking you to provide a reproduction case so we can diagnose and understand what has to be done, if it is a bug in the image, in asciidoctor, or something else.

- As you can see in https://github.com/asciidoctor/docker-asciidoctor/issues/214, the version of Docker can be a cause of troubles: what version are you using?
- Can you run the commands `docker run --rm --entrypoint=java asciidoctor/docker-asciidoctor:1.8.1 -version` and `docker run --rm --entrypoint=java asciidoctor/docker-asciidoctor -version` and share the output with us?
- Can you share an adoc file that triggers the error for you (even a simplified one) please?

**Comment by mojavelinux** [comment]
This feels eerily similar to #214. Is it really the case that Docker is broken that badly that it can't find files?

**Comment by dduportal** [comment]
Hello @aowss , without any feedback or example to reproduce your issue, I'm closing the issue assuming that the latest release `1.9.0` that fixes the issue #214 will also fix your issues.

Please feel free to reopen with a reproduction example (or at least give us some details on your environment so we can investigate) if you still have the same issue with the tag `1.9.0`

**Comment by aowss** [comment]
I did some tests, and i wanted to confirm that with version `1.7.0` and `1.9.0`, it works but with `1.8.0` it doesn't.
So the bug introduced in `1.8.0` has been fixed imho.
